src/pages/questions/ProductInformation.jsx

```diff
@@ -122,7 +123,7 @@ const ProductInformation = ({ question }) => {
         labelPlacement="end"
       />
       {!hideImages && productData?.images && (
-        <Stack spacing={2} direction="row" flexWrap="wrap">
+        <Stack direction="row" flexWrap="wrap" gap='1em'>
```

Comment:
Possibly better to use [Grid](https://mui.com/material-ui/react-grid/)?

Reply:
The idea was to fix image's width and let them wrap when needed, but using Grid with a number of images per row varying with breaking points sounds interesting

Reply:
I merged because the main topic was fixing the shortcuts. I open an issue for using Grid